package.json

```diff
@@ -33,15 +34,23 @@
       "git add"
     ]
   },
+  "jest": {
+    "roots": [
+      "./tests/migration"
```

Comment:
Could we do side-by-side tests instead? ie. put the tests in `app/` beside thing component we're testing, just named thing.spec.js or thing.test.js?
I think jests default `testMatch` setting works for this.